andrea/js/app.js

```diff
@@ -0,0 +1,67 @@
+// buttons //
+var signUp = document.querySelector(".navbar-sign");
+var logIn = document.querySelector(".navbar-login");
+
+var create_btn = document.querySelector(".create-btn");
```

Comment:
inconsistent variable naming, either use `camelCase` (as in `logIn`) or `snake-case` (as in `create-btn`), although camelCase is preferred